src/tests/ftest/util/ior_test_base.py

```diff
@@ -227,6 +228,44 @@ def check_subprocess_status(self, operation="write"):
                 self.job_manager.process, self.ior_cmd):
             self.fail("Exiting Test: Subprocess not running")
 
+    def check_pool_free_space(self, expected_scm=None, expected_nvme=None,
+                              timeout=30):
+        """Check pool free space with expected value.
+        Args:
+            expected_scm (int, optional): pool expected SCM free space.
+            expected_nvme (int, optional): pool expected NVME free space.
+            timeout(int, optional): time to fail test if it could not match
+                expected values.
+        """
+        if not expected_scm and not expected_nvme:
+            self.fail("at least one space parameter must be specified")
+        if self.pool is None:
+            self.fail("pool is none")
+        done = 0
+        scm_fs = 0
+        ssd_fs = 0
+        self.pool.connect()
+        start = time()
+        while time() - start < timeout:
+            sleep(1)
+            scm_index, ssd_index = 0, 1
```

Comment:
Optional - If we're setting these statically, better practice is to set before the `while` loop